Lib/fontTools/qu2cu/cli.py

```diff
@@ -31,6 +32,12 @@ def _font_to_cubic(input_path, output_path=None, **kwargs):
         glyph.draw(pen)
         glyf[glyphName] = ttpen.glyph(preserveTopology=False)
 
+    if stats["4"]:
```

Comment:
that looks weird.. what's '4'?

Reply:
4 means "cubic", 3 means "quadratic".

Reply:
I'll add a comment.

Reply:
because in cu2qu.ufo and also in cu2quPen the stats dict keys are number of off-curves, I think, not number of total control points including start and end points... See how they do len(curve) - 2

Reply:
whereas your Qu2CuPen seems inconsistent in that regard

Reply:
I thought it's just at printing time that we do the -2. Let me find that code.

Reply:
You are correct. I'll fix.

Reply:
Ah. The pen is correct. I should use 2 instead of 4 here.

Reply:
> Ah. The pen is correct. I should use 2 instead of 4 here.

are you sure the pen is correct? It's not doing - 2

https://github.com/fonttools/fonttools/blob/085b489012b45e55ff5eac25bf80c7781311c8ba/Lib/fontTools/pens/qu2cuPen.py#L52-L56

Reply:
No. Let me clean this up.

Reply:
so to decide whether you end up with any cubics, basically, you're checking that the stats contain an entry for the count of curves/splines with 2 offcurves. However, is it conceivable that the Qu2CuPen keeps all the quad splines that have 2 off-curves as quadratic and doesn't actually convert them to cubics? In which case, you are setting the glyphDataFormat to 1 unnecessarily?